eo-runtime/src/test/eo/org/eolang/string-tests.eo

```diff
@@ -73,3 +73,7 @@
       "Hello"
       42
 
+[] > supports-escape-sequences
```

Comment:
@andreoss maybe it's good to add a similar test to `char-tests.eo`?

Reply:
@yegor256 Done.